release/release_gold.sh

```diff
@@ -317,4 +320,47 @@ function test_boms {
 	fi
 }
 
+function update_release_info_date {
+	if [[ "${_PRODUCT_VERSION}" != *q* ]] ||
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 3)" -eq 0 ]] ||
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1)" -lt 2024 ]]
+	then
+		lc_log INFO "Skipping the release info update."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	if [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1)" -eq 2024 ]] &&
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 2 | tr -d q)" -le 2 ]] &&
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 3)" -le 11 ]]
+	then
+		lc_log INFO "Skipping the release info update."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	if [[ ! " ${@} " =~ " --test " ]]
+	then
+		lc_cd "${_PROJECTS_DIR}/liferay-portal-ee"
+
+		local product_group_version="$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1,2)"
+
+		local quarterly_release_branch_name="release-${product_group_version}"
+
+		git branch --delete "${quarterly_release_branch_name}" &> /dev/null
+
+		git fetch --no-tags upstream "${quarterly_release_branch_name}":"${quarterly_release_branch_name}" &> /dev/null
+
+		git checkout "${quarterly_release_branch_name}" &> /dev/null
+
+		sed -e "s/release.info.date=.*/release.info.date=$(date -d @"${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" +"%B %-d, %Y")/" -i release.properties
+
+		git add "${_PROJECTS_DIR}/liferay-portal-ee/release.properties"
+
+		git commit -m "Updating the release info date for ${_PRODUCT_VERSION}."
+
+		git push upstream "${quarterly_release_branch_name}"
```

Comment:
@kiwm, we have very similar code in the function `prepare_next_release_branch`, please see https://github.com/liferay/liferay-docker/blob/1b2d63fe00bb495208ec9b6794d55972e65c0789/release/release_gold.sh#L115-L135.

Can you try to create a function and reuse it in both places?
```
function update_release_info_date {
...
prepare_branch_to_commit

sed -e "s/release.info.date=.*/release.info.date=$(date -d @"${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" +"%B %-d, %Y")/" -i release.properties

commit_to_branch "Updating the release info date for ${_PRODUCT_VERSION}." "${_PROJECTS_DIR}/liferay-portal-ee/release.properties"
...
}

function commit_to_branch {
	git add "${2}"

	git commit -m "${1}"

	git push upstream "${quarterly_release_branch_name}"
}

function prepare_branch_to_commit {
	lc_cd "${_PROJECTS_DIR}/liferay-portal-ee"

	local quarterly_release_branch_name="release-${product_group_version}"

	git branch --delete "${quarterly_release_branch_name}" &> /dev/null

	git fetch --no-tags upstream "${quarterly_release_branch_name}":"${quarterly_release_branch_name}" &> /dev/null

	git checkout "${quarterly_release_branch_name}" &> /dev/null
}


```
Thanks